Assets/Scripts/Battle/BattleSystem/BattleParticipant.cs

```diff
@@ -9,7 +11,16 @@ public abstract class BattleParticipant : MonoBehaviour
 
     public BattleAction currentAction { get; protected set; }
 
-    public bool Dead { get { return currentHp <= 0; } }
+    [SerializeField]
+    protected List<ElementType> Weaknesses = new List<ElementType>();
```

Comment:
This will require us to set the weaknesses and resistances for every enemy. Based on the current design conversations, it would be simpler to assign an enemy to a single element type, then use a hard-coded type chart to determine weaknesses/resistances, like how Pokémon does.